platform-includes/getting-started-run-alternative/javascript.node.mdx

```diff
@@ -0,0 +1,17 @@
+<SignInNote />
+
+```javascript {tabTitle:CommonJS}
+require('./instrumentation.js')
+
```

Comment:
I would argue that we should not honor this escape hatch an entire code block.

Considerations:
- People might do both --require AND the import
- It might be confusing
- We _really_ do not want people to do that unless absolutely necessary and a dedicated code block might be promoting it.

Reply:
I have a different opinion: We _definitely_ should to show this, because there will def. be people out there who do not have the (easy) capability to adjust the code their app is run in production. Lots of devs do not have direct access to their deployment infrastructure etc, making it _much_ harder for them to get started with using sentry. We can call out why the other solution is preferable and lead with that, but should still show how you can get it to run otherwise. There is really nothing wrong with the instrument-on-top approach for require.js.

Reply:
actually, we need to keep this anyhow and add a note that this is _definitely_ needed for node <18 - no other way to run stuff there, I think?

Reply:
> --require is supported since v1.6.0

Ah, sweet! Good to know 🚀 

> We were just thinking that people will skip over reading the leading text and just copy paste code snippets around, ending up with both --require and a top level import.

You need to read this anyhow, as the text instructs you to create another file etc anyhow - this is not as easy as copy-pasting a snippet into your app.js anyhow, I'd argue. If you don't read this, it'll not work anyhow.

I'd just like to argue that while _for us_ it is fine and easy to "just add a CLI flag when running the node app", there are _loads_ of devs out there for whom this is not _as_ trivial, and (as stated before) even more that simply can't do this themselves. Why should we not document how they can still use Sentry even if they can't easily adjust this themselves? I don't really see the upside of "hiding" this away? It's not even that problematic if this is double-initialized, it should work just fine even if somebody does that. And we could even guard against this, if we actually need it.

Reply:
Yea you're right. I can definitely see some orgs disallowing passing flags to node, or like you said, people not even having the ability to change how things are run.

Will keep it, but maybe give it its own heading — as is, it's easy to not see that this is an alternative way of running it, wdyt?